pkg/services/cloudmigration/cloudmigrationimpl/authapistub.go

```diff
@@ -0,0 +1,67 @@
+package cloudmigrationimpl
```

Comment:
Based on previous conventions that I noticed for fake/mock services, shouldn't this be located in the `authapi` package under a subpackage `fakeauthapi` or something like it?

Reply:
I agree this should be moved in authapi package, unless you need some implementation specific to what cloud migration testing would need but this one seems quite generic right now.

Reply:
fixed in https://github.com/grafana/grafana/pull/96581/commits/ec2ba2d8ce6fef5ea7c0f68827cf5bca74290dee
Agree. I just left it there because the gcomstub was also there. since this is the only service that use these apis, it is kind of the same, but I agree that is better in the same package as the real implementation

Reply:
actually, moved it to fake sub-package